Library/Homebrew/extend/os/mac/formula_cellar_checks.rb

```diff
@@ -58,36 +56,10 @@ def check_python_framework_links(lib)
     EOS
   end
 
-  def check_linkage
-    return unless formula.prefix.directory?
-    keg = Keg.new(formula.prefix)
-    checker = LinkageChecker.new(keg, formula)
-
-    if checker.broken_dylibs?
-      audit_check_output <<-EOS.undent
-        The installation was broken.
-        Broken dylib links found:
-          #{checker.broken_dylibs.to_a * "\n          "}
-      EOS
-    end
-
-    # only check undeclared deps for standard installations.
```

Comment:
👍 that check is useful. The explanation message isn't great for that check either, but the check itself is useful at catching things that should have been revision bumped at some point but were not, in particular.